hack/dind-cluster.sh

```diff
@@ -327,7 +327,7 @@ function nodes-are-ready() {
   # Skip the SDN node since nothing is intended to be scheduled on it.
   local node_count=$(${DOCKER_CMD} exec -t "${MASTER_NAME}" bash -c "\
 KUBECONFIG=${DEPLOYED_CONFIG_ROOT}/openshift.local.config/master/admin.kubeconfig \
-oc get nodes | grep -v ${SDN_NODE_NAME} | grep -v NotReady | grep Ready | wc -l")
+oc get nodes | grep -v SchedulingDisabled | grep -v NotReady | grep Ready | wc -l")
```

Comment:
Yeah, ok. So this will print a list of all ready, schedulable nodes:

    oc get nodes --template '{{range $item := .items}}{{if not .spec.unschedulable}}{{range .status.conditions}}{{if eq .type "Ready"}}{{if eq .status "True"}}{{printf "%s\n" $item.metadata.name}}{{end}}{{end}}{{end}}{{end}}{{end}}'